packages/@aws-cdk/aws-autoscaling/lib/mixed-autoscaling-group.ts

```diff
@@ -0,0 +1,47 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import { CfnAutoScalingGroup } from './autoscaling.generated';
+
+/**
+ * Base mixed instances policy for autoscaling group
+ */
+export interface MixedInstancesPolicyBase {
+  /**
+   * The instances distribution to use
+   */
+  readonly instanceDistribution: CfnAutoScalingGroup.InstancesDistributionProperty
```

Comment:
No L1 types at the L2 level please.

Also, I wonder if it might make more sense to inline these properties here. We try not to nest too much.